packages/terra-horizontal-divider/src/HorizontalDivider.scss

```diff
@@ -0,0 +1,13 @@
+@import './variables';
+@import './mixins';
+
+:local {
+  .divider {
+    border: 0 solid;
+    border-color: var(--terra-horizontal-divider-border-color, #b2b5b6);
+    border-top: 1px solid #dedfe0;
+    border-width: var(--terra-horizontal-divider-border-width, 1px);
+    height: 0;
+    margin: 0;
+  }
```

Comment:
I think we only need to specify border-width, border-color, and border-style? So something like this maybe:
```
:local {
  .divider {
     border-color: var(--terra-horizontal-divider-border-color, #b2b5b6);
     border-style: solid;
     border-width: var(--terra-horizontal-divider-border-width, 1px);
     height: 0;
     margin: 0;
   }
```

Reply:
Updated https://github.com/cerner/terra-core/commit/5fb27563217e211a5ec708f6dc65da66fb369f31#diff-23c60d19275a3c865f09b7cdc50dbcd8R2